cpp/rrtmgp_const.h

```diff
@@ -27,6 +27,32 @@ using FOView = Kokkos::Experimental::OffsetView<T, Kokkos::LayoutLeft, Device>;
 
 template <int Rank, typename ExecutionSpace=Kokkos::DefaultExecutionSpace>
 using MDRangeP = Kokkos::MDRangePolicy<ExecutionSpace, Kokkos::Rank<Rank> >;//, Kokkos::Iterate::Right, Kokkos::Iterate::Left> >;
+
+// Copied from EKAT
+template <typename View>
```

Comment:
I don't see these two types used anywhere...Do you need them? Am I just not seeing them?

Reply:
Oh, I added these before I realized that I could just use the same type. I will remove.